store/Cargo.toml

```diff
@@ -18,24 +18,25 @@ doctest = false
 [dependencies]
 anyhow = { version = "1.0" }
 clap = { version = "4.3" , features = ["derive"] }
-deadpool-sqlite = { version = "0.5", features = ["rt_tokio_1"]}
+deadpool-sqlite = { version = "0.6", features = ["rt_tokio_1"]}
 directories = { version = "5.0" }
 figment = { version = "0.10", features = ["toml", "env"] }
 miden-crypto = { git = "https://github.com/0xPolygonMiden/crypto", branch = "next" }
```

Comment:
May make sense to add `default-features = false` here.

Reply:
Added `default-features = false` and `features = ["std"]`. I'm not sure if that is better.

Note that `std` is required, because errors in the server are using `anyhow`, and that library depends on `error::Error` from `std` (it is not available on `core` only).